omps/api/v1/about.py

```
@@ -5,10 +5,12 @@
 
 from flask import jsonify
 
+import pkg_resources
 from omps import __version__ as version
 from . import API
 
 
 @API.route("/about", methods=('GET',))
 def about():
-    return jsonify(version=version)
+    courier_version = pkg_resources.get_distribution('operator-courier').version
```

May this fail? Shouldn't we catch an exception and return `unknown` version?

---

This will fail if `operator-courier` is missing. In which case, I think it's ok, to let this fail unhandled.

---

ok